apps/settings/sub_menu/exam_mode_controller.cpp

```diff
@@ -112,7 +113,11 @@ GlobalPreferences::ExamMode ExamModeController::examMode() {
   GlobalPreferences::ExamMode mode = GlobalPreferences::sharedGlobalPreferences()->tempExamMode();
   if (GlobalPreferences::sharedGlobalPreferences()->isInExamMode()) {
     // If the exam mode is already on, this re-activate the same exam mode
+    #if ION_SIMULATOR_FILES
+    mode = GlobalPreferences::ExamMode::Off;
+    #else
```

Comment:
I think these lines should be removed, because this pull request is marked as ready for review

Reply:
Yes, but it is marked as ready to merge…

Reply:
Indeed because I think it can be a good idea to add the possibility to disable the exam mode in simulators ^^ The exam mode in simulator doesn't make any sense in the first place because you can always leave the simulator and open others app so I don't see the problem by adding the option

Reply:
Yes, but if we look #131, it is built as a simulator, but it runs on the calculator…

Reply:
Yes, I agree with Yaya-Cout